modules/camera/camera_device.py

```diff
@@ -8,24 +8,33 @@
 import cv2
 import numpy as np
 
+import picamera2
+
 
 class CameraDevice:
     """
     Wrapper for camera.
     """
 
-    def __init__(self, name: "int | str", save_nth_image: int = 0, save_name: str = "") -> None:
+    def __init__(
+        self, name: "int | str", save_nth_image: int = 0, save_name: str = "", use_pc2: bool = False
```

Comment:
name is no longer a required parameter, make use_picamera a required parameter.
Also, please rename use_pc2 to use_picamera or something not as abbreviated

Reply:
I could use some clarification on what this means